database.js

```diff
@@ -0,0 +1,9 @@
+const db = 'mongodb://localhost/lizardboard'
+const url = 'mongodb://127.0.0.1:27017/lizardboard';
```

Comment:
Let's make this upper case `URL`, and maybe name it to be more specific (`DATABASE_URL` perhaps?)